pandas/tests/frame/test_iteration.py

```diff
@@ -159,4 +159,4 @@ def test_sequence_like_with_categorical(self):
             str(s)
 
         for c, col in df.items():
-            str(s)
+            str(s)  # pylint: disable=undefined-loop-variable
```

Comment:
I'm not sure if this s is intended to be here, but it looks like the function `test_sequence_like_with_categorical` isn't called anywhere anyway.